source/fundamentals/collations.txt

```diff
@@ -0,0 +1,539 @@
+are the same. Consider the following Canadian French words:
+
+- cote
+- coté
+- côte
+- côté
+
+When using binary collation, MongoDB sorts them in the following order:
+
+.. code-block:: none
+   :copyable: false
+
+   cote
+   coté
+   côte
+   côté
+
+When using the Canadian French collation, MongoDB sorts them in a different
+order as shown below:
+
+.. code-block:: none
+   :copyable: false
+
+   cote
+   côte
+   coté
+   côté
+
+
+.. _collations-specify:
+
+How to Specify Collations
+-------------------------
+
+MongoDB supports collations on most :doc:`CRUD operations </fundamentals/crud>`
+and aggregations. For a complete list of supported operations, see the
+:manual:`Operations that Support Collations </reference/collation/#operations-that-support-collation>`
+server manual page.
+
+You can specify the locale code and optional variant in the following string
+format:
+
+.. code-block:: none
+   :copyable: false
+
+   "<locale code>@collation=<variant code>"
+
+Specify a valid locale in the ``locale code`` placeholder and optionally a
+variant in the ``variant code`` placeholder. The following example specifies
+the "de" locale code and "phonebook" variant code:
+
+.. code-block:: none
+   :copyable: false
+
+   "de@collation=phonebook"
+
+If you do not need to specify a variant, omit everything after the locale
+code as follows:
+
+.. code-block:: none
+
+   "de"
+
+For a complete list of supported locales, see our server manual page on
+:manual:`Supported Languages and Locales </reference/collation-locales-defaults/#supported-languages-and-locales>`.
+
+The following sections show you different ways to apply collations in
+MongoDB:
+
+- :ref:`Collection <collations-collection>`
+- :ref:`Index <collations-index>`
+- :ref:`Operation <collations-operation>`
+
+.. _collations-collection:
+
+Collection
+~~~~~~~~~~
+
+You can set a default collation when you create a collection. When you
+create a collection with a specified collation, all supported operations
+that scan that collection apply the rules of the collation.
+
+The following snippet shows how to specify the "en_US" locale collation
+when creating a new collection called ``items``:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
+   :language: java
+   :dedent:
+   :start-after: start createCollectionOptions
+   :end-before: end createCollectionOptions
+
+To check whether you created the collation successfully, retrieve a list
+of the indexes on that collection as follows:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
+   :language: java
+   :dedent:
+   :start-after: start listIndexes
+   :end-before: end listIndexes
+
+The output of your code should contain the following:
+
+.. code-block:: none
+   :copyable: false
+
+   { ...
+     "collation": { "locale": "en_US", ... }
+     ...
+   }
+
+.. note::
+
+   You can only assign a default collation to a collection when you
```

Comment:
Nit: It's a little weird reading this to have a note like this that refers to the very next section on the very same page.

Reply:
Ok, I'll move it to a spot prior to the example. Let me know if that looks better to you, once committed.